pkgs/by-name/op/openmpi/package.nix

```diff
@@ -240,29 +238,23 @@ stdenv.mkDerivation (finalAttrs: {
         ))
         (lib.concatStringsSep "\n")
       ]}
-      # A symlink to $\{lib.getDev pmix}/bin/pmixcc upstreeam puts here as well
-      # from some reason.
-      moveToOutput "bin/pcc" "''${!outputDev}"
 
       # Handle informative binaries about the compilation
-      for i in {prte,ompi,oshmem}_info; do
```

Comment:
Could you explain these removals? Maybe they should belong to a separate commit that would explain them as well? I assume they apply to both Linux and Darwin platforms. If not, don't change these commands for Linux.